AnnService/src/Core/SPANN/SPANNIndex.cpp

```diff
@@ -203,7 +203,7 @@ namespace SPTAG
 
             m_index->SearchIndex(*p_queryResults);
             
-            if (m_pQuantizer.get() != m_index->m_pQuantizer.get()) { p_queryResults->SetTarget(p_queryResults->GetTarget(), m_index->m_pQuantizer); }
+            //if (m_pQuantizer.get() != m_index->m_pQuantizer.get()) { p_queryResults->SetTarget(p_queryResults->GetTarget(), m_index->m_pQuantizer); }
```

Comment:
Let's just delete, and if we need to support different head and SSD quantizer, do it in a different PR

Reply:
sure